argo_jupyter_scheduler/utils.py

```diff
@@ -73,9 +86,30 @@ def gen_cron_workflow_name(job_definition_id: str):
     return f"job-def-{job_definition_id}"
 
 
-def gen_output_path(input_path: str):
-    p = Path(input_path)
-    return str(p.parent / "output.ipynb")
+def gen_default_output_path(input_path: str):
+    # The initial filename before we can get access to the timestamp. Has the
+    # "unknown" suffix to make it visible in the logs and to avoid clashing with
+    # the input filename.
+    return gen_output_path(input_path, "unknown")
+
+
+def gen_default_html_path(input_path: str):
+    # The initial filename before we can get access to the timestamp. Has the
+    # "unknown" suffix to make it visible in the logs and to avoid clashing with
+    # the input filename.
+    return gen_html_path(input_path, "unknown")
+
+
+def gen_output_path(input_path: str, timestamp: str):
+    # See create_output_filename in jupyter-scheduler
+    basename = os.path.splitext(input_path)[0]
+    return f"{basename}-{timestamp}.ipynb"
+
+
+def gen_html_path(input_path: str, timestamp: str):
+    # See create_output_filename in jupyter-scheduler
+    basename = os.path.splitext(input_path)[0]
+    return f"{basename}-{timestamp}.html"
```

Comment:
This is where we create filenames that need to match the upstream `create_output_filename` from jupyter-scheduler. I think we should refactor this later to use the upstream `create_output_filename` function and make these accept an `int` instead of a string timestamp. But I have no time to do this change right now.

Reply:
Realized it's an easy fix to do, so made the change.